src/components/Search/FuseSearch.js

```diff
@@ -0,0 +1,22 @@
+import Fuse from 'fuse.js';
+import productList from '../../data/productsList';
+
+const options = {
+  // isCaseSensitive: false,
```

Comment:
these options can also be part of a different file, something called **fuseConfig**